eng/scripts/run_tests.ps1

```diff
@@ -8,16 +8,16 @@ Param(
 )
 
 $ErrorActionPreference = 'Stop'
```

Comment:
As an alternative you could set the legacy command line parsing https://github.com/Azure/azure-sdk-for-go/blob/main/eng/common/scripts/git-branch-push.ps1#L42

Reply:
Ohh yes this is that issue manifest all over again. I think for new scripts we should just get used to the new behavior and not rely on implicit globals like this.